### ISSUE TITLE
Sliders (skipped element) become less smooth

### ISSUE THREAD

**Issue by antonio1475** [zanna-37/hass-swipe-navigation#26]
I just noticed something: hass-swipe-navigation makes the sliders not as smooth as the originals. I had realized that Mushroom sliders weren't as smooth, but I had not linked it to hass-swipe-navigation at all.

When testing the new sliders, I thought "wow, they are so smooth". And after the update to 1.7.0 to ignore them, not anymore and it clicked.
I reverted to 1.6.5 to confirm, and indeed. I also removed the add-on and tested Mushroom sliders, and smooth again.

**It is a very small difference, and only really noticeable on slow movements and for sure not everyone will even notice**, and it's a decent compromise for the received funcitonality, but is this a known side effect?

Let's see if the screen recording allows to show it (if reduced to 30FPS, it's not noticeable).

https://user-images.githubusercontent.com/63130724/206449085-a2b59533-5597-4809-ab4b-8e60ae5d596a.mp4

https://user-images.githubusercontent.com/63130724/206449098-8fb516fe-94ff-4d5f-89ef-692d95edc38c.mp4

Thanks again!

**Comment by antonio1475** [comment]
Addition: it seems that the culprit (at least partial) is iOS Battery Saver mode (which I have always on).

I've been doing some more testing and _sometimes_ the slider is smooth (with Battery Saver on), so I looked outside of the add-on or HA, and disabling iOS Battery Saver mode seems to consistently make the slider smooth, even with hass-swipe-navigation.

So I guess with Battery Saver on, the smoothness depends on how much CPU/GPU iOS allows for HA at each time?
I _suppose_ hass-swipe-navigation is adding an overhead, so without the add-on it was smooth anyway (with Battery Saver mode). 

I don't know, just thinking "out loud" 😅

**Comment by zanna-37** [comment]
That's interesting. The biggest overhead should be on `touchstart`, when the finger first touches the display. In fact, it's here that I check what element the finger is touching and compare it to all the exceptions.
During `touchmove` (finger is dragging), I only update some variables and nothing more. So it should be instantaneous.

To sum up, I can expect an (very low) overhead on `touchstart`, but not on `touchmove`.

Can you investigate a little more? To precisely pinpoint the cause?

**Comment by antonio1475** [comment]
@zanna-37 I stand corrected. hass-swipe-navigation has nothing to do with this.

[iOS Battery Saver mode reduces FPS to ~30](https://www.google.com/search?q=ios+battery+saver+30fps&oq=ios+battery+saver+30fps).

Even with hass-swipe-navigation removed, same thing happens. Since it's not 100% of the time, I got confused and thought it was due to hass-swipe-navigation, but it's not true. I should have Googled more beforehand, I mixed things!

My apologies!
